plusminus/plusminus.py

```diff
@@ -1295,7 +1346,7 @@ class Operators:
 
         special_exponents_opns_map = {
             "⁻¹": (lambda x: 1 / x),
-            "⁰": (lambda x: x ** 0),
```

Comment:
Please leave as-is. Doing the '**' operation on `x` will raise an exception if `x` is of a type (such as a set or a str) that does not support '**', which will eventually bubble up to the caller.